### ISSUE TITLE
Team member unable to see Github repo in repo view

### ISSUE THREAD

**Issue by domudall** [dependencyci/support#33]
A team member signed up to Dependency CI is unable to access a project, even though they are signed up to the repo in question. They are not, however, a member of the organisation on Github.

Dependency IO view:
![image](https://cloud.githubusercontent.com/assets/524482/16807703/61c1f770-4911-11e6-9bb4-710cdafbb97c.png)

Project settings:
![image](https://cloud.githubusercontent.com/assets/524482/16807731/7a497926-4911-11e6-88bb-f5e47de69199.png)

Project is `ConnectedVentures/f8-transactional-api`

**Comment by domudall** [comment]
Also realised this is a duplicate 😞 apologies. #5

**Comment by andrew** [comment]
Yeah it's not clear right now but they need to enable private repo usage by going to https://dependencyci.com/enable_private

**Comment by andrew** [comment]
Will leave this open so I remember to make it less confusing!
